Dockerfile

```diff
@@ -1,4 +0,0 @@
-# Before running this run "npm run build"
```

Comment:
How can it be nowified if it’s not in a container?

Reply:
now doesn't do containers anymore

Reply:
sorry mom, im about to break master